docs/hf_hub.rst

```diff
@@ -111,3 +111,40 @@ returns the model output for a given input. It can be used as::
 
 In the above code snippet, ``res`` will be a :class:`numpy.ndarray` containing
 the model's output.
+
+.. _hf_hub_gradio:
+
+Easily build user interfaces to your scikit-learn models
+--------------------------------------------------------
+`gradio <https://gradio.app/>`__ is a python library that lets you create interfaces on your model.
+It has a class called :class:`Interface` that lets you create application
+interfaces to your machine learning models.
+``gradio`` is integrated with skops, so you can load an interface with only one 
+line of code. You need to initialize an interface, call load method with
+your repository identifier (your user name and name of the model repository)
+prepended with "huggingface/" will load an interface for your model. The interface
+has a dataframe input that takes samples and a dataframe output to return 
+predictions. It also takes the example in the repository that is previously
+pushed with skops. Calling :func:`gr.Interface.launch()` will launch your application.
+
+.. code:: python
+
+    import gradio as gr
+    repo_id = "scikit-learn/tabular-playground"
+    gr.Interface.load(f"huggingface/{repo_id}").launch()
```

Comment:
This works for me:

```python
gr.Interface.load("models/scikit-learn/tabular-playground").launch()
```

Reply:
same it worked for me as well, however, we have this script: https://skops.readthedocs.io/en/latest/auto_examples/plot_hf_hub.html#sphx-glr-auto-examples-plot-hf-hub-py in which we create an example model and repository automatically, and I previously put interface building at the end of it where gradio tried to retrieve that model and it did, but was failing to read the example input. I also checked that model specifically (that was pushed) and could build the interface myself. I feel like I'm missing something here 😅

TLDR; gradio can't create the UI to the model pushed through notebook when used inside the same notebook, but it can create UI when I call the model from somewhere else.

Reply:
Gradio currently reads examples from README so that we can also load in models that were uploaded without skops. If you can read in the model yourself eventually, that makes me thing the examples are added to the README automatically after a delay?

Reply:
@freddyaboulton that might be the case (?)